src/template.mligo

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: 2021 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
+// Replace TEMPLATE_H with something unique corresponding to your
+// variant. For example, if your variant is called MyBaseDAOVariant you
+// can change TEMPLATE_H to MYBASEDAOVARIANT_H so this section reads
+//
+// #if !MYBASEDAOVARIANT_H
+// #define MYBASEDAOVARIANT_H
+
+#if !TEMPLATE_H
+#define TEMPLATE_H
+
+//
+
+#include "types.mligo"
+
+// -----------------------------------------------------------------
+// Proposal check
+//
+// This should implement the proposal_check logic of the variant and should
+// verify whether a proposal can be submitted.  It checks 2 things: the
+// proposal itself and the amount of tokens frozen upon submission.  It allows
+// the DAO to reject a proposal by arbitrary logic and captures bond
+// requirements
+// -----------------------------------------------------------------
+
+let proposal_check (_, _ : propose_params * contract_extra) : unit = unit
+
+// -----------------------------------------------------------------
+// Decision_lambda
+//
+// The decision lambda is executed based on a successful proposal.  It has
+// access to the proposal, can modify `contractExtra` and perform arbitrary
+// operations.
+// -----------------------------------------------------------------
+
+let decision_lambda (input : decision_lambda_input)
```

Comment:
It seems a bit odd that this is still called `decision_lambda` now, given that it's no longer technically a lambda.

What do you think about renaming it?

Reply:
Have renamed it to `decision_callback`.